test/Faker/Provider/ru_RU/PersonTest.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+namespace Faker\Test\Provider\ru_RU;
+
+use Faker\Generator;
+use Faker\Provider\ru_RU\Person;
+use PHPUnit\Framework\TestCase;
+
+class PersonTest extends TestCase
+{
+    /**
+     * @var Generator
+     */
+    private $faker;
+
+    public function setUp()
```

Comment:
Visibility should be `protected` here, but we can - with the right rules - automatically fix.